src/ussd_tb_check.js

```diff
@@ -691,40 +695,226 @@ go.app = function () {
         question: text,
         error: error,
         accept_labels: true,
-        choices: [new Choice("state_show_results", $("See Results"))],
+        choices: [new Choice("state_display_arm_message", $("See Results"))],
       });
     });
 
-    self.states.add("state_show_results", function (name) {
```

Comment:
Shouldn't we still show the original message if study is not active yet and arm is empty?

Reply:
@erikh360 Which one is the original message ?